### PR TITLE
redirect eventlet.net to readthedoc

### PR DESCRIPTION
**Pull request by 4383**
Remove outdated documentation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,162 +1,10 @@
...
-<li><a class="reference external" target="_blank" href="irc://chat.freenode.net/#eventlet">IRC channel</a></li>
-<li><a class="reference external" target="_blank" href="https://www.openhub.net/p/eventlet">Open Hub stats</a></li>
-</ul>
-        </div>
-      </div>
-    </div>
-    <script src="//yandex.st/highlightjs/7.3/highlight.min.js"></script>
-    <script>hljs.initHighlightingOnLoad();</script>
-  </body>
+<html>
```
Comment (itamarst) [inline index.html:2]: Maybe also some text with a link, for people with JavaScript disabled?

**Review by itamarst — APPROVED** [review]
Feel free to merge with or without fix.

File: index.html (hunk 1 above)
Comment (itamarst) [reply index.html:2]: Although I guess this is temporary until DNS gets updated so not a permanent solution anyway.

Comment (4383) [reply index.html:2]: Will submit a follow up patch.

**Pull request merged**